bin/csv_from_oai.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+# OAI client setup
+client = OAI::Client.new(
+  "http://oai.adventistdigitallibrary.org/OAI-script",
+  headers: { from: "jeremy@scientist.com" },
```

Comment:
Does this need to be dynamic, or will Jeremy's email forever be in the code base?

Reply:
Job security?!? 😁

Reply:
email was made dynamic